.github/workflows/regression.yml

```diff
@@ -186,15 +110,20 @@ jobs:
           cache-to: type=gha,mode=max
           file: regression/Dockerfile
           builder: ${{ steps.buildx.outputs.name }}
+          outputs: type=docker,dest=/tmp/baseline-image.tar
```

Comment:
If I understand GitHub Actions correctly, a potential issue with using `/tmp` on a self-hosted runner is that data in `/tmp` may persist over multiple workflow runs, whereas any data in `${{ runner.temp }}` is limited to the lifetime of the workflow run, as discussed in, for example, https://github.com/actions/setup-elixir/issues/24.

Reply:
Spot on.

Reply:
Addressed in 24977f50d